README.md

```diff
@@ -153,93 +186,92 @@ This can be used to represent all kinds of errors. It can be for errors that hap
 
 
-## <a name="host"></a> Host fields
+### <a name="host"></a> Host fields
 
-All fields related to a host. A host can be a physical machine, a virtual machine, and also a Docker container.
+Host fields provide information about the host. A host can be a physical machine, a virtual machine, or a Docker container.
 
-Normally the host information is related to the machine on which the event was generated / collected but also can be used differently if needed.
+**TBD. Originally said "physical machine, virtual machine, AND a Docker container." Should that be OR instead of AND**
+Normally host fields are related to the machine that generated/collected the event, but they can be used differently if needed.
 
 
 | Field  | Description  | Type  | Multi Field  | Example  |
 |---|---|---|---|---|
-| <a name="host.timezone.offset.sec"></a>`host.timezone.offset.sec`  | Timezone offset of the host in seconds.<br/>Number of seconds relative to UTC. In case the offset is -01:30 the value will be -5400.  | long  |   | `-5400`  |
-| <a name="host.name"></a>`host.name`  | host.name is the hostname of the host.<br/>It can contain what `hostname` returns on Unix systems, the fully qualified domain name or also a name specified by the user. It is up to the sender to decide which value to use.  | keyword  |   |   |
-| <a name="host.id"></a>`host.id`  | Unique host id.<br/>As hostname is not always unique, this often can be configured by the user. An example here is the current usage of `beat.name`.  | keyword  |   |   |
+| <a name="host.timezone.offset.sec"></a>`host.timezone.offset.sec`  | Timezone offset of the host in seconds.<br/>Number of seconds relative to UTC. Example: If the offset is -01:30, the value will be -5400.  | long  |   | `-5400`  |
+| <a name="host.name"></a>`host.name`  | Identifier of the host.<br/>It can contain what `hostname` returns on Unix systems, the fully qualified domain name, or a name specified by the user. The sender decides which value to use.  | keyword  |   |   |
+| <a name="host.id"></a>`host.id`  | Unique host id.<br/>As hostname is not always unique, this often can be configured by the user.<br/>TBD: "This" = What speficially? <br/>Example: The current usage of `beat.name`.  | keyword  |   |   |
 | <a name="host.ip"></a>`host.ip`  | Host ip address.  | ip  |   |   |
 | <a name="host.mac"></a>`host.mac`  | Host mac address.  | keyword  |   |   |
-| <a name="host.type"></a>`host.type`  | This is the type of the host.<br/>For Cloud providers this can be the machine type like `t2.medium`. Or it vm, container for example or something user defined.  | keyword  |   |   |
-| <a name="host.os.platform"></a>`host.os.platform`  | Operating system platform (e.g. centos, ubuntu, windows).  | keyword  |   | `darwin`  |
+| <a name="host.type"></a>`host.type`  | TBD</br>Type of host.<br/>For Cloud providers this can be the machine type, such as `t2.medium`. Or it vm, container for example or something user defined.  | keyword  |   |   |
```

Comment:
Let's move this also to a separate PR for discussion.

Reply:
This is flagged "TBD" because I couldn't figure out what we're trying to say here:
`Or it vm, container for example or something user defined.`
  I'll take a crack at it. If I guess incorrectly, we can explore in another PR.